### DISCUSSION TITLE
Generalist Repository Selection Flowchart

### DISCUSSION THREAD

**Discussion by pg05** [NIH-GREI/grei-pm#169] [Feedback]
### Please select one or more topics you'd like to provide feedback on.

- [ ] GREI Event (Webinar, Workshop, Community Call, etc)
- [ ] GREI Year 3 Objective, Task, or Deliverable
- [X] GREI Resources (Comparison Chart, Selection Flowchart, etc)
- [ ] Other (please specify)

### If applicable, select the GREI Year 3 Objective, Task, or Deliverable.

- [ ] Community Engagement (All units)
- [ ] Desirable  Repository Characteristics
- [ ] Consistent Metadata
- [ ] Search & Browse
- [ ] Analytics & Reporting
- [ ] Use Cases
- [ ] Connecting Digital Objects
- [ ] QA/QC
- [x] Training & Community Engagement

### Questions or Comments

```bash
1. Have you used the Generalist Repository Selection Flowchart to identify a repository to use? 
2. What other characteristics should be included? 

Share your thoughts here! No need to create git issues. 

About the Flowchart:
GREI created the Flowchart (https://zenodo.org/records/11105430) to help researchers and those providing research support services decide which repository is best to share research outputs. 

Learn about the work: https://github.com/NIH-GREI/grei-pm/issues/35
```

**Comment by LucaFoschini** [comment]
Thank you again to the GREI consortium for your efforts in guiding the user community to select the most suitable repositories for data sharing—this is such a valuable resource.

As I mentioned in yesterday’s public feedback session, my understanding of the flowchart (and previously, the comparison chart) is that it aims to provide guidance across all nine NIH-listed Generalist Repositories, as noted in the link below, rather than focusing solely on the seven within the GREI consortium:

https://sharing.nih.gov/data-management-and-sharing-policy/sharing-scientific-data/generalist-repositories

If that’s the intent, we’d like to offer an updated resource that includes Synapse by Sage Bionetworks for your consideration (COI Disclosure: I’m the President of Sage):

https://acrobat.adobe.com/id/urn:aaid:sc:VA6C2:eaea9fee-d356-42f4-acce-1dea550ec656?viewer%21megaVerb=group-discover

Thank you for considering this addition. I look forward to continuing the conversation and contributing to this important work.

**Reply by nicipfeiffer** [reply]
Thank you, Luca, for attending the feedback session and providing this updated resource for GREI to consider. We will review and follow up soon on the outcome.

**Comment by shlake** [comment]
I'll put all my thoughts here in one comment. Let me know if you would like issues created.

- This suggestion might go along with comment above. The flow chart needs a "Title". If this is for NIH specifically, then that should be made clear.
- Use the following URL for the Dataverse guide link (in the chart on right): https://guides.dataverse.org/en/latest/user/index.html It goes to the latest version for the Dataverse software that is on Harvard's Dataverse repository.
- For the 2nd branch (institutional repository), would be better to say "Check with your institution's repository" and not just say "use". There may be reason not to use it.
- Third square (**visit this guide**), the URL should point to the "latest" version of the comparison chart for Generalist Repo, this one: https://doi.org/10.5281/zenodo.3946719
- At this point in the flow chart (third square), you may want to say that the next steps in the workflow helps you choose (and is based on the comparison chart) - this link **(visit)** is just for reference to use later?
- Someone during the feedback call mentioned that the "peer review" choice seemed out of place, not defined. I agree.
- Confusion about the square "complying with FAIR". The choices after "Yes" are just things one might like to do in a repository, not necessarily FAIR. Plus the grey box at the top right of the chart says "each repository shares the following features.... FAIR compliant". So by using a GREI repository, the researcher is already complying with FAIR, right?
- Suggestion for renaming this square.... "Do you need specific support?" (there might be a better word than my suggestion "support"?).
- Will people know what **deposit curation** means?
- There should be a key for what "$" (the dollar sign means).
- What is the difference between "Paid curation services" under additional fees vs. the curation services (with the "$" above on the FAIR line)? I would have expected all "$" repos on the FAIR line (both curation boxes) would appear at this additional services box "Paid Curation Services".
- The license chart needs the repository icons (symbols) at the top. Having the names is fine, but to make things consistent the repo icons need to be added (at the top?).

**Reply by pg05** [reply]
Many thanks for attending the call and sharing your thoughts with us, Sherry. This list is extremely helpful and our team will reference it. There is no need to create git issues.

**Comment by bryanmgee** [comment]
Some thoughts from me in addition to those that I made on the call:

- I agree with @shlake on how a number of terms (offered services) are unclear here. I would also submit 'Research environment' and 'Independent review panel (IRP)', which are exclusive to Vivli, as other examples. For me (either as a librarian or as a scientist), it's unclear what they are and whether these are exclusive to Vivli because it is not a true generalist like the others (i.e. these are features for clinical data), or, these are features that could be in any generalist platform and simply happen not to be found in the others. That may make it more confusing for someone who isn't a clinical researcher (but may assume Vivli is a viable option for them to consider based on its inclusion in GREI). The hyperlinking of these could easily be confused with a formatting error, especially when other hyperlinked content is the typical blue colour. I would suggest just considering removing these two Vivli-exclusive features since the vast majority of people who need to find a home for their data and don't have a domain-specific repo are not clinical researchers. If Synapse is added, maybe these are not unique features, but then the hyperlinking is also not possible.
- +1 for Sherry's comment about usage of IRs as well; even at institutions that may be a member of one or more of the GREI repos, the benefits of membership aren't always changing the experience a researcher would have submitting versus if their institution was not a member (e.g., sometimes a membership is more designed for librarians/admins who want to track outputs or have someone else maintain the infrastructure, like OSF or figshare memberships vs. having an APC waived, like in Dryad memberships).
- In the vein of potential misinterpretation of formatting, in Increased Storage, the Zenodo box that says 'ZE' instead of a '$' like every other repo could be misconstrued as a typo. I know what it means (after three takes) because I'm pretty familiar with Zenodo's policies and their case-by-case-for-over-50GB policy specifically, but that not something the average researcher using this guide will know or take away from this; I'm not even sure all librarians would know that. It doesn't help that the preceding green circle says "Additional fees _may_ apply" and then that applies to every feature for every listed repo except Zenodo's increased storage.
- In the same vein as the 'peer review' comments, I actually think it would be more useful to indicate whether a repository supports anonymization of unpublished content shared privately (to support "blind" or "anonymized" review).   
- Whether embargoes are permitted (I think of embargoes as withholding data beyond the publication of an associated article/termination of a funded grant; the existing GREI comparison chart makes it sound more like during peer review).
- The more I look at it, I think it could be confusing to have the 'Anonymize data before submission to repository' in the same format as the other "boxed" items, which are all services, when this is a best-practice/pseudo-requirement for submission, not a service. I think it would be better as a bubble, perhaps merged with the larger green one to the right.      
- I am sure this must have come up in previous conversations, but I found it a bit odd that features of figshare+, which are still the same org, are not uniformly incorporated here (e.g., >50 GB of storage, more licensing options); the main caveat to figshare+ - that someone has to pay - applies for certain features in other repos as well (e.g., going above 50 GB in Dryad costs money, and Dryad is listed up to 300 GB here in addition to '$' for Increased Storage).
- It is a little ambiguous whether 'Other' licenses is a catch-all for literally any other term/condition/IP protection, established licenses not previously listed or only found in one repository, or custom terms that don't need to be an existing license (like what Dataverse offers). I am also unclear (again speaking as a former Dryad curator) what 'Other' means for Dryad, which does not accept anything other than CC0-compatible materials; the Zenodo integration that permits additional licensing for mediated deposits to Zenodo isn't a license but a feature process. 
- I would recommend some kind of visual differentiator between software and data licenses, especially when they are not grouped by attribute (e.g., CC BY-NC is separated from CC BY and CC0). This could be shading the rows for one type, another column, etc.
- Excessive whitespace around the margins? The dimensions of this don't look like a standard size for print or display that would warrant it. It could be helpful to retain as a buffer if additional info is added, like defining some of the more ambiguous service offerings.
- Possibly some kind of additional boxes or a table listing integrations for upload with common cloud storage (Dropbox, Box, Google Drive, etc.), "repositories" (GitHub, Bitbucket, etc.), and transfer mechanisms (e.g., API, Globus). This is a very common question researchers have, and those that have more/all of these tend to get a lot more recommendations at my institution. 

Thanks y'all!

**Reply by pg05** [reply]
Thank you for attending the call and your additional feedback, Bryan. We greatly appreciate it.
